Sources/SwiftKuery/Table.swift

```diff
@@ -138,42 +139,51 @@ open class Table: Buildable {
         }
         return result
     }
-    
-    /// Function that returns the SQL CREATE TABLE query as a String for this TABLE.
-    /// - Returns: A String representation of the table create statement.
-    /// - Throws: QueryError.syntaxError if statement build fails.
+
+    /**
+     Function that returns the SQL CREATE TABLE statement for this table in string fromat.
+     ### Usage Example: ###
+     In this example we define a simple table named exampleTable and then print its description
+     ```swift
+     class ExampleTable: Table {
+         let tableName = "ExampleTable"
+         let name = Column("name", String.self)
+     }
+     let examples = ExampleTable()
+     let description = try examples.description(connection: getConnection(from: postgresPool))
+     print(description)
+     // Prints CREATE TABLE ExampleTable (name text)
+     ```
+     
+     /// - Returns: A String representation of the table create statement.
```

Comment:
If these are inside the multiline comment they don't need comment lines here and don't display properly in Jazzy.